src/app/file-browser/components/file-list/file-list.component.ts

```diff
@@ -124,8 +125,10 @@ export class FileListComponent implements OnInit, AfterViewInit, OnDestroy, HasS
     @Optional() private drag: DragService,
     private renderer: Renderer2,
     public device: DeviceService,
-    private ngZone: NgZone
+    private ngZone: NgZone,
+    private api: ApiService
```

Comment:
Why is this added here? This seems like a remnant from earlier work. I think we can remove it and the ApiService import.